backend/test_optimization.py

```
@@ -0,0 +1,119 @@
+            "Content-Type": "application/json",
+            "Accept-Encoding": "gzip"
+        }, timeout=30)
+        
+        if response.status_code == 200:
+            data = response.json()
+            size = len(json.dumps(data))
+            compressed_size = len(response.content)
+            
+            print(f"\n{'='*60}")
+            print(f"Endpoint: {endpoint}")
+            print(f"Minimal Mode: {minimal}")
```

Ruff (0.15.7)

[error] Undefined name `minimal`

(F821)